src/app.js

```diff
@@ -113,7 +113,7 @@ if (ENABLE_PERF_LOGGING) {
       perfCounters[label] = 0
     }
     if (typeof perfTimers[label] === 'undefined') {
-      perfTimers[label] = Date.now()
+      perfTimers[label] = global.nativePerformanceNow()
```

Comment:
Where do you get this `nativePerformanceNow` from?

Reply:
It’s built into react native.